Gemfile

```diff
@@ -2,5 +2,10 @@ source 'https://rubygems.org'
 
 gem 'rake'
 
+group :test, :development do
+  gem 'rspec'
+  gem 'pry'
```

Comment:
Maybe use [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug)? Would you guys be in favor of starting to test it out. It seems to be better than pry